modules/appeals_api/app/controllers/appeals_api/higher_level_reviews/v0/higher_level_reviews_controller.rb

```diff
@@ -10,8 +10,10 @@ class HigherLevelReviewsController < AppealsApi::V2::DecisionReviews::HigherLeve
     skip_before_action :validate_icn_header
     skip_before_action :new_higher_level_review
     skip_before_action :find_higher_level_review
+    skip_before_action :validate_json_format
 
     before_action :validate_icn_parameter, only: %i[download]
+    prepend_before_action :validate_json_body, if: -> { request.post? }
```

Comment:
This is prepended because `validate_json_format` and `validate_json_body` set `@json_body`, which the decision reviews controllers depend on (and the appeals controllers still inherit from those decision reviews controllers). Skipping `validate_json_format` messes with the order of the `before_action`s such that I needed to prepend the replacement, since others depend on `@json_body`.

Reply:
This is a noice compromise.